server/libs/infra/src/repositories/asset.repository.ts

```diff
@@ -129,6 +136,19 @@ export class AssetRepository implements IAssetRepository {
         };
         break;
 
+      case WithoutProperty.FACES:
+        relations = {
+          faces: true,
+        };
+        where = {
+          resizePath: IsNull(),
+          isVisible: true,
+          faces: {
+            assetId: IsNull(),
+            personId: IsNull(),
+          },
+        };
+
```

Comment:
Missing `break`

Reply:
Yeah if called with `WithoutProperty.FACES` it will fall through to default and throw an error. This function is only used for the queue, so the recognize faces job won't work. Uploads use the `handleRecognizeFaces` function and can still work fine